packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -51,4 +232,69 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
 
     return result
   }
+
+  /**
+   * Requests attestations for a phone number
+   * @param {string} phoneNumber The phone number for which to request attestations for
+   * @param {number} attestationsRequested The number of attestations to request
+   * @param {string} token The token with which to pay for the attestation fee
+   */
+  async request(phoneNumber: string, attestationsRequested: number, token: CeloToken) {
+    const phoneHash = PhoneNumberUtils.getPhoneHash(phoneNumber)
+    const tokenAddress = await this.kit.registry.addressFor(token)
+    return wrapSend(
+      this.kit,
+      this.contract.methods.request(phoneHash, attestationsRequested, tokenAddress)
+    )
+  }
+
+  /**
+   * Reveals the phone number to the issuer of the attestation on-chain
+   * @param {string} phoneNumber The phone number which requested attestation
+   * @param {string} issuer The address of issuer of the attestation
+   */
+  async reveal(phoneNumber: string, issuer: string) {
+    const publicKey = await this.contract.methods.getDataEncryptionKey(issuer).call()
+
+    if (!publicKey) {
+      throw new Error('Issuer data encryption key is null')
+    }
+
+    const encryptedPhone: any =
+      '0x' +
+      ECIES.Encrypt(
+        // @ts-ignore
```

Comment:
what's the error here?

Reply:
Same as above, due to the incorrect type, I made the same solution like you suggested